README.md

```
@@ -92,6 +110,52 @@ Then run
 npx cap sync
 ```
 
+## Optional Configuration
+
+To use certain features of this plugin, you will need to add the following permissions/keys to your native project configurations.
+
+### Android
+
+In your `android/app/src/main/AndroidManifest.xml`:
+
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)